examples/pipe/dev/system.clj

```diff
@@ -3,18 +3,33 @@
   development.
   The `system/start` and `system/stop` functions are required by the
   `user` namespace and should not be called directly."
-  (:require [clojure.string :as str]
-            [clojure.tools.logging :as log]
-            [clojure.java.shell :refer [sh]]
-            [jackdaw.admin :as ja]
-            [pipe]))
+  (:require [jackdaw.admin :as ja]
+            [jackdaw.client :as jc]
+            [pipe]
+            [clojure.edn :as edn])
+  (:import (java.util UUID)))
 
-(def system nil)
+(defonce system (atom nil))
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;; Configs ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
 (defn kafka-admin-client-config
   []
   {"bootstrap.servers" "localhost:9092"})
 
+(def producer-config
```

Comment:
Added configs for the Kafka producer and consumer to make it easier for newcomers to run the tutorial/walkthough.